packages/eslint-plugin-internal/configs/base-node.js

```diff
@@ -0,0 +1,18 @@
+const ecmaVersion =
+  require('../../../tsconfig-bases/node-cjs.json').compilerOptions.target.substring(2);
```

Comment:
The idea behind this is to avoid duplicating ECMA version between tsconfigs vs. ESLint configs.